pkg/lockfile/parse.go

```diff
@@ -8,6 +8,10 @@ import (
 	"strings"
 )
 
+// To avoid linter error (goconst) on some parsers.
+// String used when pkgName is not present
```

Comment:
nit: I think we should expand a bit more on this

```suggestion

// Represents when a package name could not be determined while parsing.
// Currently, parsers are expected to omit such packages from their results.
```